test/e2e/serial/e2e_serial_test.go

```diff
@@ -22,13 +22,22 @@ import (
 	"time"
 
 	. "github.com/onsi/ginkgo/v2"
+	ginkgo_reporters "github.com/onsi/ginkgo/v2/reporters"
 	. "github.com/onsi/gomega"
 
+	qe_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
+
 	serialconfig "github.com/openshift-kni/numaresources-operator/test/e2e/serial/config"
 	_ "github.com/openshift-kni/numaresources-operator/test/e2e/serial/tests"
 )
 
+var afterSuiteReporters = []Reporter{}
+
 func TestSerial(t *testing.T) {
+	if qe_reporters.Polarion.Run {
```

Comment:
where should this argument be set? I see it is false by default

Reply:
The module is adding the flags at init time. I prefer a more explicit approach, but still this is a common pattern in the golang world:
https://github.com/kubevirt/qe-tools/blob/main/pkg/ginkgo-reporters/README.md
https://github.com/kubevirt/qe-tools/blob/main/pkg/ginkgo-reporters/polarion_reporter.go#L37